morphos/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: morphos
+category: files
+name: Morphos Server
+version: "0.1.1"
+tagline: Self-hosted file converter server
+description: >-
+  Today we are forced to rely on third party services to convert files to other formats. This is a serious threat to our privacy, if we use such services to convert files with  highly sensitive personal data. It can be used against us, sooner or later.
+
+  Morphos server aims to solve the problem mentioned above, by providing a self-hosted server to convert files privately. The project provides an user-friendly web UI.
+
+  For now, Morphos only supports images. Documents will be added soon.
+releaseNotes: >-
+  PDF support is finally here! Easily convert to images to PDF files.  Otherwise is also possible.
```

For the initial release on umbrelOS, we can leave the release notes blank.

```suggestion
releaseNotes: ""
```

---

Gone, too!